//Fix for bug #80631 Any ResultSet.getString return garbled result wi…

neil4dong opened the pull request:
http://bugs.mysql.com/bug.php?id=80631
//Fix for bug #80631 Any ResultSet.getString return garbled result with json type data

// Fix chinese(or other non-ascii encoding) garbled in [mysql 5.7 JSON type]
// Use UTF-8 encoding , because mysql use utf8mb4 for JSON type store ;

SORRY for formatted this file ,Eclipse did that.

neil4dong committed:
//Fix for bug #80631 Any ResultSet.getString return garbled result with json type data
// Fix chinese(or other non-ascii encoding) garbled in [mysql 5.7 JSON type]
// Use UTF-8 encoding , because mysql use utf8mb4 for JSON type store ;

neil4dong commented:
I have sent OCA by the instructions, please confirm it

mysql-admin commented:
Hello,
you have confirmed that you have already sent you OCA by instructions, however I am not able to find that, can you please send it directly to me at lenka(dot)kasparova(at)oracle(dot)com? I will make sure it's processed asap.
thank you,
Lenka

neil4dong commented:
Hi!
 have you received my OCA . I've sent that few days ago.
lenka(dot)kasparova(at)oracle(dot)com

mysql-admin commented:
Hi Neil,
unfortunately I have not received that, from which address have you sent that? maybe it was deleted by our server, can you please try to send that to my personal address (lenka.kasparova(at)gmail(dot)com) as well as to oracle-ca_us(at)oracle.com? I am so sorry for such complications but it's first time after long time I am not receiving emails...
Thank you,
Lenka

neil4dong commented:
I have sent the OCA from (email_dsl(at)163(dot)com) to (lenka.kasparova(at)gmail(dot)com). 

Thank you.

neil4dong committed:
//Fix for bug #80631 Any ResultSet.getString return garbled result with json type data
// Fix chinese(or other non-ascii encoding) garbled in [mysql 5.7 JSON type]
// Use UTF-8 encoding , because mysql use utf8mb4 for JSON type store ; (reverted from commit b3ff97aa610d853025e83f75991fffbd37458ff3)

neil4dong committed:
//Fix for bug #80631 Any ResultSet.getString return garbled result with json type data
// Fix chinese(or other non-ascii encoding) garbled in [mysql 5.7 JSON type]
// Use UTF-8 encoding , because mysql use utf8mb4 for JSON type store ;

mysql-admin commented:
re-open

neil4dong commented:
I confirm the code being submitted is offered under the terms of the OCA, and that I am authorized to contribute it.
